src/server/routes/accounts.js

```diff
@@ -1,48 +1,59 @@
-let express = require('express');
-let router = express.Router();
-let Account = require('../models/Account');
+const express = require('express');
+const router = express.Router();
+const queries = require('../database/queries');
 
 router.use((req, res, next) => {
   // TODO verify request is authenticated before proceeding
-    // Below is working properly...
+    // Below is not working properly...
 
   // if(req.isAuthenticated()) {
   //   next();
   // } else {
   //   console.log('You are not logged in');
   //   res.json({Error: 'Authorization error'});
   // }
-  console.log(req.session);
+  // console.log(req.session);
   next();
 });
 
 router.get('/:uid/balance', (req, res) => {
-  Account.getBankAccounts(req.params.uid)
-  .then(accounts => {
-    res.send(accounts)
+  queries.getAccountBalance(req.params.uid)
+  .then(balance => {
```

Comment:
for each of these, what happens if .catch is called instead? Consider error handling for each place you use .then.
I recommend changing each of these handlers to be async, and then use `try { } catch { }` around the `await`'d calls.